vtex-apps/store-image
# Fix 'experimentalPreventLayoutShift' prop not being passed down from ImageList to Image components

#### What problem is this solving?

The `experimentalPreventLayoutShift` prop supported by the `Image` component was not exposed for users to configure when they were using the `list-context.store-image` block. 

This PR exposes this configuration to users.

#### How to test it?

Go to this [Workspace](https://victormiranda2--carrefourbr.myvtex.com/), scroll down a bit and use any of the `slider-layout` blocks that are in the page. All of them use the block `list-context.image-list` with the `experimentalPreventLayoutShift` feature turned OFF (we did not allow users to turn it OFF before).

Just to check the issue this feature being always on was causing, go to [Carrefour](https://carrefourbr.myvtex.com/) and play with the `slider-layout`s. You should see a strange flicker in the items beside the sliders, as described in https://github.com/vtex-apps/store-discussion/issues/428. 

#### Related to / Depends on

Closes https://github.com/vtex-apps/store-discussion/issues/428.

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/BsQAVgY6ksvIY/giphy.gif)
